### ISSUE TITLE
Extended logging

### ISSUE THREAD

**Issue by reinco** [Klebert-Engineering/zswag#68]
Implement extended logging on all steps executed by zswag, including timings.
In case zswag is waiting on some (service) response, zswag should do periodic logging of what it is doing as well.
The behavior should be parameterized (disabled by default).

Reason: currently zswag api acts as a black box. If a service connected to is responding slow or not at all it is unclear why this is happening. 
By adding a more verbose mode such issues can be better investigated.

**Comment by josephbirkner** [comment]
Thanks for raising the issue! Would spdlog be suitable for you?

**Comment by reinco** [comment]
I do not really know. I saw it has cyclic logging, which will help to prevent getting excessively large log files.

My hope was that with appropriate logging I could see why sometimes instantiating the OAClient python class does not return.
In such cases I did not see any log from OAServer on our service side, whereas the json output of the service is visible in a browser.
Sometimes the issue was resolved by restarting LiveLab, which happened frequently on Windows but never on Linux.
